default/python/universe_generation/specials.py

```diff
@@ -122,14 +130,15 @@ def distribute_specials(specials_freq, universe_objects):
             specials_timer.start("Randos")
             random_sys = random.choice(systems_needing_specials.values())
             member = random.choice(list(random_sys))
-            candidates.append(member[0])
+            (obj, sys, n) = member
```

Comment:
`(obj, sys, n)` - > `obj, sys, n`  no need parenthesis here
`sys` shadows python builtin module, please rename to `systems`. 
`n` -> `specials_count`